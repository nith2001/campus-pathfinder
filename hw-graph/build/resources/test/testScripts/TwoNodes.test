# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
ListNodes graph1
AddNode graph1 n2
ListNodes graph1

# Print the nodes in the graph and their outgoing edges
ListNodes graph1