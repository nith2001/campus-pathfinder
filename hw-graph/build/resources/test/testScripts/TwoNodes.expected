# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1
graph1 contains: n1
added node n2 to graph1
graph1 contains: n1 n2

# Print the nodes in the graph and their outgoing edges
graph1 contains: n1 n2